services/resources/Dockerfile

```diff
@@ -1,15 +1,9 @@
-FROM ubuntu
-ENV PYTHONUNBUFFERED 1
-
-RUN apt-get update
-
-RUN apt-get install -y python3-pip
+FROM python:3
 
 RUN mkdir /code
 WORKDIR /code
 
 ADD . /code/
```

Comment:
If you're using volumes control in your `docker-compose.yml`, there's no need for copying the entire code to the image.